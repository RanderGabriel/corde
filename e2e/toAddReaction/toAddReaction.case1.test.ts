import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("tests should be sucessfull", async () => {
  const command = Utils.buildCommandWithConfigPath("toAddReaction", "bot_case1.test.ts");
  const results = await cli.exec(command);
  expect(results.statusCode).toEqual(0);
  expect(results.stdout).toMatchSnapshot();
});
