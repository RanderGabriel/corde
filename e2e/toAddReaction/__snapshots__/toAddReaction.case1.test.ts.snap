// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests should be sucessfull 1`] = `""`;
